recipes-nodes/haproxy/haproxy.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "HAProxy"
+LICENSE = "HAPROXY's license - 2006/06/15"
+FILESEXTRAPATHS:prepend := "${THISDIR}:"
+
+SRC_URI += "file://init \
+            file://haproxy.cfg.mustache"
+
+INITSCRIPT_NAME = "haproxy"
+INITSCRIPT_PARAMS = "defaults 98"
```

Comment:
Do you want to run this earlier than this order or is it the same run level as rbuilder?

Reply:
HAProxy should run before orderflow-proxy which is at runlevel 99. The rest is not important.